phone.py

```diff
@@ -6,18 +6,23 @@ def __init__(self, phone_letters):
         self.phone_letters = phone_letters
 
     def validate_phone(self):
-        """
-          Return false if a phone is not valid.
-          A valid phone number should match these characteristics:
-          - It should start with +.
-          - It shouldn't be smaller than 15 characters.
-          - Aside from the + no other special character should be permitted (i.e: #,@,%,etc).
-        """
+        #"""
+          #Return false if a phone is not valid.
+          #A valid phone number should match these characteristics:
+          #- It should start with +.
+          #- It shouldn't be smaller than 15 characters.
+          #- Aside from the + no other special character should be permitted (i.e: #,@,%,etc).
+        #"""
+        phone_doesnt_start_with_plus = Phone("+13463335555")
+        phone_greater_than_15_characters = Phone("+13463335555321412342134")
+       	phone_with_special_characters = Phone("!~#$%^&*@|}{")
```

Comment:
@Anahis259 please remove this. I am curious, why do you think that you have to add these lines?

Reply:
Chicos yo me encargo de hacer los cambios donde marque "Aquí"

Reply:
Aquí

Reply:
ya modifique las ultimas 3 correcciones, las subo...

Reply:
changes made

Reply:
new changes made